### PR TITLE
Added [kube] and [gke] sections to config.toml.

### PR DESCRIPTION
**Pull request by mwhittaker**
Thanks Robert for the suggestion.

### CODE REVIEW

File: weaver.toml [hunk 1]
```diff
@@ -11,11 +11,26 @@ binary = "template"
...
-# `weaver multi deploy`.
+# `weaver multi deploy`. See serviceweaver.dev/docs.html#multiprocess for more
+# information about the multiprocess deployer.
 [multi]
 listeners.lis = {address="localhost:9000"}
+
+# The [kube] section configures the behavior of the app when deployed via
+# `weaver kube deploy`. See github.com/serviceweaver/weaver-kube for
+# more information about the kubernetes deployer.
+[kube]
```
Comment (rgrandl) [inline weaver.toml:28]: I think for the [kube] deployer, they have to specify the image name too?

File: weaver.toml [hunk 2]
```diff
@@ -11,11 +11,26 @@ binary = "template"
...
 
 # The [multi] section configures the behavior of the app when deployed via
-# `weaver multi deploy`.
+# `weaver multi deploy`. See serviceweaver.dev/docs.html#multiprocess for more
+# information about the multiprocess deployer.
 [multi]
 listeners.lis = {address="localhost:9000"}
+
+# The [kube] section configures the behavior of the app when deployed via
+# `weaver kube deploy`. See github.com/serviceweaver/weaver-kube for
```
Comment (rgrandl) [inline weaver.toml:26]: can we add a TODO to point out to serviceweaver.dev/docs.html#kube instead once we add the kube documentation in the docs?

**Review by rgrandl — APPROVED** [review]

File: weaver.toml (hunk 1 above)
Comment (mwhittaker) [reply weaver.toml:28]: Good catch. Done.

File: weaver.toml (hunk 2 above)
Comment (mwhittaker) [reply weaver.toml:26]: I don't want to add the TODO here since the user will see it when they run `gonew`, but I'll keep a mental TODO.

**Review by mwhittaker — COMMENTED** [review]
Thanks Robert!

**Pull request merged**